src/main/java/org/openstreetmap/atlas/geography/atlas/AtlasMetaData.java

```diff
@@ -105,6 +159,14 @@ public AtlasSize getSize()
         return new HashMap<>(this.tags);
     }
 
+    @Override
+    public int hashCode()
+    {
+        final int sizeHash = this.getSize().hashCode();
+        return Objects.hash(Integer.valueOf(sizeHash), Boolean.valueOf(this.original),
```

Comment:
Just checking - data version field was left out intentionally here?

Reply:
No it should be there haha. Since I check against it in equals(), I should be using it in the hash computation. Good catch.